web_monitoring/cli/cli.py

```diff
@@ -204,7 +236,7 @@ def run(self):
 
             self.handle_record(record, retry_connection_failures=True)
 
-        self.wayback.close()
+        # self.wayback.close()
```

Comment:
It makes sense that if an `adapter` is passed in to `__init__` it's the user's job to close it (as written now) because that enables sharing it between instances for parallel use. But if `adapter=None` and we create our own for this instance, should we close it here?

Reply:
Ah yeah, that’s a good point!